src/examples/java/com/amazonaws/crypto/examples/FileStreamingExample.java

```diff
@@ -49,22 +49,21 @@
     public static void main(String[] args) throws IOException {
         srcFile = args[0];
 
-        // In this example, we'll pretend that we loaded this key from
-        // some existing store but actually just generate a random one
+        // In this example, we generate a random key. In practice, 
+        // you would get a key from an existing store
         SecretKey cryptoKey = retrieveEncryptionKey();
 
-        // Convert key into a provider. We'll use AES GCM because it is
-        // a good secure algorithm.
+        // Create a JCE master key provider using the random key and an AES-GCM encryption algorithm
         JceMasterKey masterKey = JceMasterKey.getInstance(cryptoKey, "Example", "RandomKey", "AES/GCM/NoPadding");
 
-        // Instantiate the SDKs
+        // Instantiate the SDK
         AwsCrypto crypto = new AwsCrypto();
 
-        // Create the encryption context to identify this ciphertext
+        // Create an encryption context to identify this ciphertext
         Map<String, String> context = Collections.singletonMap("Example", "FileStreaming");
 
-        // The file might be *really* big, so we don't want
-        // to load it all into memory. Streaming is necessary.
+        // Because the file might be to large to load into memory, we use 
+        // streaming, then encrypt the file stream.
```

Comment:
__note:__ Get @bdonlan to review this wording to make sure there's not a better way of describing what Java is doing with the file.